python/tk_desktop2/websockets/requests/local_file_linking/pick_file.py

```diff
@@ -60,3 +51,65 @@ def execute(self):
                 files.append(f)
 
         self._reply(files)
+
+
+class PickFilesOrDirectoriesWebsocketsRequest(PickFileOrFilesWebsocketsRequest):
+    """
+    Command to pick multiple files or directories.
+
+    Pops up a modal file selector dialog and lets the user choose
+    one or more files or directories.
+    Part of the local file linking feature set.
+
+    Request syntax::
+
+        No parameters are defined for this command
+
+    Expected response::
+
+        A list of file paths as strings.
+    """
+
+    def __init__(self, connection, id, parameters):
+        """
+        :param connection: Associated :class:`WebsocketsConnection`.
+        :param int id: Id for this request.
+        :param dict parameters: Command parameters (see syntax above)
+        :raises: ValueError
+        """
+        super(PickFilesOrDirectoriesWebsocketsRequest, self).__init__(
+            connection, 
+            id, 
+            pick_multiple=True
+        )
+
+
+class PickFileOrDirectoryWebsocketsRequest(PickFileOrFilesWebsocketsRequest):
+    """
+    Command to pick multiple files or directories.
```

Comment:
This is single-file/directory picking, right?

Reply:
well spotted! fixed!